app/controllers/spotlight/pages_controller.rb

```diff
@@ -85,6 +89,20 @@ def update_all
       redirect_back fallback_location: spotlight.exhibit_dashboard_path(@exhibit), notice: notice
     end
 
+    def clone
+      destroy_pre_existing_page_for_locale
```

Comment:
Happy to do this in a different way.  This is basically what gives us the `Recreate` functionality.